packages/inertia/src/inertia.js

```diff
@@ -236,6 +249,11 @@ export default {
         }
         return this.setPage(response.data, { visitId, replace, preserveScroll, preserveState })
       }).then(() => {
+        const errors = this.resolveErrors(this.page)
+        if (Object.keys(errors).length > 0) {
+          fireErrorEvent(errors)
+          return onError(errors)
+        }
```

Comment:
What if `resolveErrors` does not return an object?

Reply:
Then that should lead to a fix in the user's `resolveErrors` implementation 😅

Reply:
Yeah, exactly. It's basically on the developer to ensure the `resolveErrors()` method always returns an object. And I fully expect most Laravel/Rails developers to never have to set this, and can simply reply on the default behaviour (which an empty object if no errors are present).